docs/narr/viewconfig.rst

```diff
@@ -494,6 +494,16 @@ configured view.
 
   .. versionadded:: 1.4a3
 
+``is_authenticated``
+
+  This value, if specified, should be either ``True`` or ``False``.  If it is
+  specified and is ``True``, the request must be for an authenticated user,
+  as determined by the :term:`security policy` in use.  If it is specified and
+  ``False``, the associated view callable will match only if the request does
+  not have an authenticated user.
```

Comment:
```
If it is specified and its value is ``True``, only a request from an
authenticated user, as determined by the :term:`security policy`
in use, will satisfy this predicate.

If it is specified and its value is ``False``, only a request from
a user who is not authenticated will satisfy this predicate.
```

The `is_authenticated` predicate is applicable to both routes and views, and this version makes it copy-pasteable into route config. I like less maintenance!

Also I don't think it is correct to say that a "view callable will match", but maybe "predicate will match" or "predicate is satisfied". I took a shot at it, but maybe @mmerickel can wordsmith it better. Whatever is correct in Pyramid terminology.

The sentences should be merged into one paragraph. I put them separately for easier comprehension.

Reply:
Oops, that should have been `the view config will match`!  (not correct to say `the view callable will be invoked`, there are other predicates)

Reply:
I have kept the doc as it was (not copy-pastable between route and view doc), to follow the existing style.

Reply:
Yeah, the predicate docs are suboptimal. There's an [open issue](https://github.com/Pylons/pyramid/issues/3597) that describes the current state, identifies the problems, and offers suggestions to improve the situation. I'd appreciate input on the best direction forward, then I'll create a PR for it.

Reply:
I’m not sure about my wording `the view config will match`, it seems a little imprecise.
(`the route will match` feels correct, if a little dry — maybe `route definition`?)

I agree now that `the predicate will match` is simple, correct and reusable so I will edit the new docs.